0000-template.md

```diff
@@ -1,7 +1,11 @@
 
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- FOLIO Issue: (leave this empty)
+- Stage: (DRAFT REVIEW, PRELIMINARY REVIEW, PUBLIC REVIEW, FINAL REVIEW)
+- RFC PRs: 
+  - DRAFT REVIEW: TBD
```

Comment:
It's a fair point.  Right now these types of changes aren't a big deal.  We've already talked about retroactively making changes to existing RFC metadata for consistency and clarity.  That's feasible because so far the number of RFCs is manageable.  My guess is that by the time we get to the point where the number of RFCs is prohibitively difficult to "fix up", we'll have gone through enough iterations or process refinement that most of these types of changes (like the number and/or naming of stages) will have already been made.

Reply:
sounds good. I am okay with leaving it as is.